src/main/java/wolox/training/models/Book.java

```diff
@@ -35,6 +36,9 @@
     @Column(nullable = false)
     private Integer pages;
 
+    @ManyToMany(mappedBy = "books")
+    private List<User> users;
```

Comment:
This should be initialized right here with, for example, a " = new ArrayList<>();"